### DISCUSSION TITLE
DNS Chaos Failure - Cannot Find DaemonIP on Node

### DISCUSSION THREAD

**Discussion by stefan-malin-db** [chaos-mesh/chaos-mesh#4203] [Q&A]
## Bug Report

**What version of Kubernetes are you using?**
v1.27

**What version of Chaos Mesh are you using?**
v2.6.1

**What did you do? / Minimal Reproducible Example**
While applying DNS Chaos using Chaos Mesh, the operation failed with an error stating it could not find the daemonIP on node gke in related Endpoints.

**What did you expect to see?**
DNS Chaos should have been applied successfully.

**What did you see instead?**
Received an error message: “Failed to apply chaos: cannot find daemonIP on node gke-olb-germany-main-24b32748-4313 in related Endpoints”

**Output of chaosctl**

**Comment by STRRL** [comment]
It seems that chaos mesh failed to connect the chaos-daemon on node `gke-olb-germany-main-24b32748-4313`.

chaos daemon is the daemonset component which is **required** to inject chaos, please check it runs well on each node.
